packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -165,7 +175,13 @@ export class AttestationService {
     }
 
     if (duty.isAggregator) {
-      setTimeout(async () => {
+      const timeout = setTimeout(async (signal = abortSignal) => {
+        this.logger.debug("AttestationService: Start waitForAggregate");
+        signal?.addEventListener("abort", () => {
```

Comment:
You could just do 
```ts
if (signal) signal.addEventListener
```
And won't need the optional chaining operator. We can figure out in a latter PR a common policy for ES versioning

Reply:
Right, but I figured we don't need the bigint literals anyway, so no reason to target ES2020 regardless.

Reply:
Also, worth mentioning that the other lodestar packages don't have es2020 target in their tsconfig either.

Reply:
I think we need to remove abort event listener as well :(

You can also pass third argument {once: true} to automatically clear if event is triggered